ConsoleApp1/Program.cs

```diff
@@ -1,9 +1,122 @@
-﻿namespace ConsoleApp1
+﻿using System.Text;
+
+namespace ConsoleApp1
 {
-	internal class Program
-	{
-		static void Main(string[] args)
-		{
-		}
-	}
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            string first = "abcde";
+            string second = "abcse";
+            string third = "What a \n gr8 park 4 sk8 ;-+=*";
+            string fourth = "HelloDDaa";
+            string fifth = "Hello and hi";
+
+
+
+            Console.WriteLine(Compare(first, second));
+            Analyze(third);
+            Console.WriteLine(Sort(fourth));
+            StringBuilder stringBuilder = new();
+            Console.WriteLine(stringBuilder.Append(Duplicate(fifth)));
+        }
+
+        static bool Compare(string a, string b)
+        {
+            if (a.Length != b.Length)
+            {
+                return false;
+
+            }
+            else
```

Comment:
block else here is useless because

Reply:
Ok. The logic is next:
1. Check for string length
2. ELSE block checks strings for equality (by chars comparing)